src/nix/flake.md

```diff
@@ -98,6 +98,7 @@ Flakes corresponding to a local path can also be referred to by a direct path re
 The semantic of such a path is as follows:
 
 * If the directory is part of a Git repository, then the input will be treated as a `git+file:` URL, otherwise it will be treated as a `path:` url;
+* If the directory path contains characters which aren't allowed in uris, then they will be automatically percent-encoded (with the exception of `#` and `?`);
```

Comment:
```suggestion
* If the directory path contains characters that aren't allowed in URIs, then they will be automatically percent-encoded (with the exception of `#` and `?`) by the command line;
```

This is still a bit vague. Can we use references with non-URI-characters in `flake.nix`?

Also please add an example to the list of examples.

Reply:
The encoding doesn't happen in the CLI, but directly during the flakeref resolution (my bad, the release notes were wrong, I fixed them). So yes.

I've added an example of a contrieved path